ui/app/AppLayouts/Profile/views/keycard/DetailsView.qml

```diff
@@ -100,7 +100,8 @@ ColumnLayout {
     }
 
     StatusListItem {
-        visible: root.keycardStore.keycardModule.keycardDetailsModel.anyOfItemsLocked
+        visible: typeof root.keycardStore.keycardModule.keycardDetailsModel !== "undefined"
```

Comment:
@borismelnik this part is changed a bit [here](https://github.com/status-im/status-desktop/blob/3f1c53dfd4f980464e8c17863cbea8c41fcdde8a/ui/app/AppLayouts/Profile/views/keycard/DetailsView.qml#L103-L104) and check is done without `undefined`, but no need anything to do here since I will anyhow need to rebase my PR and will change this.

Reply:
I can't check every each PR for understanding which part of master will be changed :) if you merged it already - I can fix :)